customizations/ncv4.sh

```diff
@@ -155,3 +110,9 @@ sudo bash -c "cat > /opt/microsoft/ncv4/graph.xml" <<'EOF'
   </graph>
 </graphs>
 EOF
+
+## Set NCCL configuration file for NCv4
+bash -c "cat > /etc/nccl.conf" <<'EOF'
+NCCL_TOPO_FILE=/opt/microsoft/ncv4/topo.xml
+NCCL_GRAPH_FILE=/opt/microsoft/ncv4/graph.xml
+EOF
```

Comment:
I see you removed `NCCL_IB_PCI_RELAXED_ORDERING=1` in newer change. Could you elaborate a bit more here?

Reply:
NCv4 doesn't have IB and so we can skip the relaxed ordering parameter for it